### PR TITLE
(prometheus) table support

### PR DESCRIPTION
**Pull request by mtanda**
I want to support non time series data in Prometheus.

https://github.com/grafana/grafana/pull/5917

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/datasource.ts [hunk 1]
```diff
@@ -256,6 +266,44 @@ export function PrometheusDatasource(instanceSettings, $q, backendSrv, templateS
     return { target: metricLabel, datapoints: dps };
   };
 
+  this.transformMetricDataToTable = function(series) {
```
Comment (torkelo) [inline public/app/plugins/datasource/prometheus/datasource.ts]: Some units for this code would be really nice :)

Comment (mtanda) [inline public/app/plugins/datasource/prometheus/datasource.ts]: I added test!

**Comment by bergquist** [comment]
This change we have to be implemented in /pkg/tsdb/prometheus as well.

**Comment by torkelo** [comment]
which will be tricky as the backend tsdb model does not support non time series yet

**Pull request merged**